portfolio.py

```diff
@@ -17,6 +45,7 @@ def normalize(self):
         """
         position_sum = sum(map(abs, self.values()))
         if position_sum == float(0):
-            raise ValueError("zero position on %s" % (self.date))
+            return True
```

Comment:
You don't need to return anything. Just divide it without any checks, and catch ZeroDivisionError later.